app/forms/concerns/course_form_helpers.rb

```diff
@@ -15,4 +15,18 @@ def clear_funding_information
   def course_subjects_changed?
     trainee.course_subject_one_changed? || trainee.course_subject_two_changed? || trainee.course_subject_three_changed?
   end
+
+  def commencement_date_valid
```

Comment:
This is great to put this in a helper, but I don't think it should be in the `CourseFormHelpers` when it's not actually used in any of the course forms.

Since it's concerned with the commencement date I'd either call is `CommencementDateHelpers` or `TraineeStartDateHelpers` - your pick!